lms/djangoapps/learner_dashboard/views.py

```diff
@@ -45,3 +45,36 @@ def view_programs(request):
     }
 
     return render_to_response('learner_dashboard/programs.html', context)
+
+
+def view_program_details(request, programId):
```

Comment:
Nitpick: every function in this module is a view (hence the name of the module). It would be more conventional to name this just `program_details`. We should clean up the other function in here.